learningequality/kolibri
# Theming fixes + couple UI fixes

## Summary

- replaces hardcoded colors with theme tokens or palette
- removes obsolete code and duplicate styles
- a couple of low-hanging fruit UI fixes and improvements

###  Noticeable UI changes (screenshots taken in the default theme)

| | Before | After |
| - | ------- | ------ |
| Active and hover states of activities buttons in the side panel of library | ![library-activity-hover-active-before](https://user-images.githubusercontent.com/13509191/150379389-05794dce-d777-4547-a3a1-941a6dd5a140.png) | ![library-activity-hover-active-after](https://user-images.githubusercontent.com/13509191/150379446-e7a8a27e-de39-4e45-861e-84ce4f8987ef.png) |
| Active and hover states of categories buttons in the side panel of library | ![library-categories-hover-active-before](https://user-images.githubusercontent.com/13509191/150379586-d28ee913-4c80-4ef8-b175-31addc35ff0f.png) | ![library-categories-hover-active-after](https://user-images.githubusercontent.com/13509191/150379616-03e50406-26d7-45b3-96ff-13e2d12934cc.png) |
| Slideshow pagination bar background | ![slideshow-pagination-before](https://user-images.githubusercontent.com/13509191/150495719-848f5735-85d3-4f2e-b015-940418455bcd.png) | ![slideshow-pagination-after](https://user-images.githubusercontent.com/13509191/150495736-f567c4ff-76ef-44d7-bfd9-b0bc5cdf2d3b.png) |
| Error message when selecting or creating a super admin account when importing a facility | ![setup-wizard-facility-import-before](https://user-images.githubusercontent.com/13509191/150495433-5c801023-b5f7-428d-a0c3-313ffc8ff6cb.png) | ![setup-wizard-facility-import-after](https://user-images.githubusercontent.com/13509191/150495467-f0c46164-40d0-4751-ad24-cb7949da6e30.png) |
| Error message on the admin credentials form when importing a facility | ![facility-import-admin-credentials-modal-before](https://user-images.githubusercontent.com/13509191/150369727-463dd53a-c4f1-4d49-b29a-683d973329ab.png) | ![facility-import-admin-credentials-modal-after](https://user-images.githubusercontent.com/13509191/150369748-e416b9e0-2f4f-40df-b434-9452f441ce5d.png) |
| Error message on the admin credentials form when importing individual user accounts |  ![import-individual-user-accounts-admin-error-before](https://user-images.githubusercontent.com/13509191/150370174-958e7f7f-7019-4b6f-b7dd-7eca27a04080.png) | ![import-individual-user-accounts-admin-error-after](https://user-images.githubusercontent.com/13509191/150370228-aa4dfe90-adf9-479c-937a-02859aeab24b.png) |
| Error message above the username input when importing individual user accounts | ![import-individual-user-accounts-error-before](https://user-images.githubusercontent.com/13509191/150370565-17ebfc82-6b08-480b-97df-bb712b9c4b06.png) | ![import-individual-user-accounts-error-after](https://user-images.githubusercontent.com/13509191/150370600-a67b191c-62b9-4b62-8636-d91649523201.png) |
| Warning message when removing a facility | ![remove-facility-warning-before](https://user-images.githubusercontent.com/13509191/150370958-61e097bb-2652-490b-a3e1-217609a6235c.png) | ![remove-facility-warning-after](https://user-images.githubusercontent.com/13509191/150370987-8c394506-b642-40eb-a02d-e025229edb7d.png) |
| Disabled search button in the side panel in library | ![search-button-disabled-before](https://user-images.githubusercontent.com/13509191/150371265-4ac49dd8-4293-4f46-bd9e-b37fd1ee4c88.png) | ![search-button-disabled-after](https://user-images.githubusercontent.com/13509191/150371291-d3db0efe-e32a-4723-bb61-bc18ce757aff.png) |
| Enabled search button in the side panel in library | ![search-button-enabled-before](https://user-images.githubusercontent.com/13509191/150371486-40e2ec79-314d-4f00-8e81-2c673f80290e.png) | ![search-button-enabled-after](https://user-images.githubusercontent.com/13509191/150371518-76e04854-b917-4831-8ca8-1db87fd4ecf2.png) |
| Beginners chip in the side panel | ![current-resource-beginners-chip-before](https://user-images.githubusercontent.com/13509191/150369201-f9507c4b-71be-45cc-bf05-73713ad28af3.png) | ![current-resource-beginners-chip-after](https://user-images.githubusercontent.com/13509191/150369237-641dd6b5-2837-4599-9ae4-d0703ae21905.png) |
| Back button in the top bar of the setup wizard for RTL languages | ![setup-wizard-back-rtl-before](https://user-images.githubusercontent.com/13509191/150372018-485610ff-4bc5-41e2-8546-18e9fcd9f506.png) | ![setup-wizard-back-rtl-after](https://user-images.githubusercontent.com/13509191/150372046-54add9e1-a3b2-48b6-933f-b7d50af91d34.png) |
| Icons color in the main side navigation menu | ![side-menu-before](https://user-images.githubusercontent.com/13509191/150495880-202b308c-54f0-4d01-9b92-b73d8c2c3b90.png) | ![side-menu-after](https://user-images.githubusercontent.com/13509191/150495903-40a14ffa-c4f1-47b0-aeac-b1e47d39e81e.png) |

## References

Resolves #8899

Related issues that are not resolved by this PR:
-  https://github.com/learningequality/kolibri-design-system/issues/295
- https://github.com/learningequality/kolibri/issues/9019
- https://github.com/learningequality/kolibri-design-system/issues/296
- https://github.com/learningequality/kolibri-design-system/issues/297
- https://github.com/learningequality/kolibri/issues/9025

## Reviewer guidance

### Code

I'd recommend reviewing commit by commit

### QA

I have no better idea than to recommend clicking through as many workflows in Kolibri you can think of and checking if you see some UI regressions, especially in regards to colors. Please test with the Kolibri default theme as well as with one of our theme plugins. I used Kolibri UN Women Common Plugin for testing [that I installed this way](https://github.com/learningequality/kolibri-un-women-common-plugin#how-can-i-install-this-plugin-for-development).

----

## Testing checklist



## PR process


## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md
